cmake/utils.cmake

```diff
@@ -91,7 +91,8 @@ function(shaderc_add_tests)
   endforeach()
 endfunction(shaderc_add_tests)
 
-# Finds all transitive static library dependencies of a given target.
+# Finds all transitive static library dependencies of a given target
+# including the target itself
```

Comment:
"possibly including"?  Because the target is only included if it is, itself, a static library.